.github/workflows/build.yaml

```diff
@@ -120,7 +111,6 @@ jobs:
         run: go mod download
 
       - name: Install ffmpeg
-        if: steps.cache-ffmpeg.outputs.cache-hit != 'true'
```

Comment:
curious why this is being removed since ffmpeg takes a long time to build

also seems unrelated to the rest of the PR

Reply:
the cache has not been working forever (screenshot below)

![image](https://github.com/user-attachments/assets/f412fb69-88b9-4044-80e3-786174a665d1)

i tried fixing it in [`3f048d0` (#3117)](https://github.com/livepeer/go-livepeer/pull/3117/commits/3f048d052147045bb684cc1258af431fb01e364d) but that led to the [next builds failing to find `protoc`](https://github.com/livepeer/go-livepeer/actions/runs/10282216718/job/28453448609?pr=3117). no idea why skipping ffmpeg install step leads to failure to find protoc (and we've observed this happen in the past too). would appreciate some insight into fixing the ffmpeg installation script if we can make use of caching there.

Reply:
Thanks for the context - I am not too familiar with the CI build actions myself. The only thing I can think of is there is a [v4 update](https://github.com/actions/cache?tab=readme-ov-file#v4) to the action

> skipping ffmpeg install step leads to failure to find protoc

weird, maybe it implicitly installs protoc somehow ... but I am not seeing anything obvious in the install script (and my own box doesn't even have it)